src/main/java/spireMapOverhaul/zones/candyland/consumables/AbstractConsumable.java

```diff
@@ -0,0 +1,22 @@
+package spireMapOverhaul.zones.candyland.consumables;
+
+import com.evacipated.cardcrawl.mod.stslib.fields.cards.AbstractCard.FleetingField;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.rooms.MonsterRoom;
+import spireMapOverhaul.abstracts.AbstractSMOCard;
+
+
+abstract public class AbstractConsumable extends AbstractSMOCard {
+    private final CardRarity realRarity;
+
+    public AbstractConsumable(String cardID, int cost, CardType type, CardRarity rarity, CardTarget target) {
+        super(cardID, cost, type, CardRarity.SPECIAL, target);
+        realRarity = rarity;
```

Comment:
So, i do this so that they don't show up like normal colorless cards, but still have individual rarities. This is because I replace common cards with common consumables, uncommon cards with uncommon consumables and rare cards with rare consumables. I thought this would be the easiest way to keep track of that, without just hardcoding it in where it matters. Also, this way they get the correct rarity banners. 
The rarityies also have a theme, commons being just simple cards, uncommons having some form of downside, and rares giving some perma scaling.

Reply:
Okay, then the issue that brings up is that as it stands, the rarities disappear when you save and reload (because they only get set when you see these cards in a shop or card reward). That's enough of a bug that something should change about this.

Here are two good options I see for resolving this:
* You could make the cards Special rarity and use `setDisplayRarity` to have them display the rarity banners
** This is probably the simplest and most straightforward approach, with the one downside being that any rarity-dependent effects won't work like people might expect (e.g. the Intrigue pack from Packmaster)
* You could make the cards Common/Uncommon/Rare but add `@NoPools` to leave them out of the normal card pool
** This is also fairly simple, with the one downside that effects that generate a random card of any color might generate these cards

There are probably other approaches too, but these came to mind since they're both nice and straightforward.

Reply:
oh, i was not aware of @NoPools, i think i will use that